src/util/ivmFactory.js

```diff
@@ -243,6 +243,11 @@ async function createIvm(code, libraryVersionIds, versionId, secrets, testMode)
     }),
   );
 
+  await jail.set('_credentials', function (...args) {
+    if (args.length == 0 || !credentials || !credentials[args[0]]) return 'ERROR';
```

Comment:
What if we send multiple args ?

Reply:
The first arg would be taken as the key, anything after that would be ignored

Reply:
Should we raise an error as it is not allowed to send multiple keys

Reply:
Also why make the function definition with (...args) then ?